caproto/threading/client.py

```diff
@@ -348,7 +341,7 @@ def received(self, bytes_recv, address):
     def command_loop(self):
         # Receive commands in 'bundles' (corresponding to the contents of one
         # UDP datagram). Match SearchResponses to their SearchRequests, and
-        # put (address, (cid1, cid2, cid3, ...)) into a queue. The receiving
+        # put (address, (name1, name2, name3, ...)) into a queue. The receiving
```

Comment:
I see why it makes sense to pass around `name` instead of elevating `cid` the way I originally did. But could we use `(name, priority)` here to remove the one-priority-per-context constraint and reinstate the ID-incrementing logic?